homework-submission/react-hyf/react-hyf-client/src/components/Homeworkweek3/Gender.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+const Gender = ({ gender }) => {
+  return (
+    <td>
```

Comment:
`<td>` needs to be inside a `table` but you are using it inside a listitem (`li`). This also gives you a warning in the dev tools `<td> cannot appear as a child of <li>`. Check your dev tools before pushing code.